sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -34,7 +35,15 @@ import com.mapbox.maps.renderer.egl.EGLCore
  * and for ensuring your use adheres to the relevant terms of use.
  */
 open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
-  private var mapController: MapController
+  internal var mapController: MapController
+    private set
+
+  private val viewAnnotationManagerDelegate = lazy { ViewAnnotationManagerImpl(this) }
```

Comment:
Why need a delegate here?

Reply:
Why do we need to init `ViewAnnotationManager` at all if user does not use it? 😄